### DISCUSSION TITLE
Localization

### DISCUSSION THREAD

**Discussion by Madis0** [standardnotes/forum#2533] [Suggestions]
Are there any plans to crowdsource localization of Standard Notes on its different platforms? 

I would prefer [Crowdin](https://crowdin.com/page/open-source-project-setup-request), however there are also [Transifex](https://www.transifex.com/pricing/) and [POEditor](https://poeditor.com/pricing/) for example. All three have free plans for open-source projects.

**Comment by atmoio** [comment]
I have received many offers from helpful users offering to localize in their language when we begin this effort. It's on my mind, but no set date just yet.

**Comment by holzi-x** [comment]
i'd like to give this one a bump. I encourage everyone to use sstandardnotes. This would be particularly easier when it were available in their languages, especially for eldery persons and for our youngest,,,

**Comment by arielsvg** [comment]
Thanks for the feedback, things are going very slowly on that front but we have made some progress :) you can rest assured that it's something we want to address at _some_ point.

**Comment by orangesunny** [comment]
Hi all,

Is there any news on the localization front? :)

As a proud Standard Notes user, I am happy to say that we in [Weblate](https://github.com/WeblateOrg/) can help with localization. In a libre and comfortable way. What do you say?

**Comment by arielsvg** [comment]
Thank you so much for the offer @orangesunny! We don't have anything more to announce at this time but we'll make sure to keep you posted here.

**Comment by dmitriy-bodroff** [comment]
@mobitar and @TheodoreChu I have similar question. Are there any plans to add custom localization by author of Listed's blog? There are a few text blocks only.

**Comment by JaspalSuri** [comment]
Currently the dev team is working on several improvements to the underlying functionality of our apps (after which we'll start adding new features to them). I don't think anything like this has been planned yet, but I will forward your feedback to the team!

**Comment by windymelt** [comment]
I'm very waiting for this feature.

**Comment by GOODJINC** [comment]
I'm Korean and I can help with localization. Can't wait for the update.
